pkg/operator/pdf/v0/convert_pdf_to_markdown_test.go

```diff
@@ -0,0 +1,62 @@
+package pdf
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/gojuno/minimock/v3"
+	"github.com/instill-ai/component/pkg/mock"
+)
+
+func TestConvertPDFToText(t *testing.T) {
+
+	test := struct {
+		name     string
+		filepath string
+	}{
+		name:     "Convert PDF file",
+		filepath: "testdata/test.pdf",
+	}
+
+	t.Run(test.name, func(t *testing.T) {
+		c := qt.New(t)
```

Comment:
Sorry, about [this last comment](https://github.com/instill-ai/component/pull/138#issuecomment-2126706733), I hadn't realised you had adopted quicktest. I saw `t.Run` and thought the tests still relied on `testing` only.

```suggestion
	c := qt.New(t)

	test := struct {
		name     string
		filepath string
	}{
		name:     "Convert PDF file",
		filepath: "testdata/test.pdf",
	}

	c.Run(test.name, func(c *qt.C) {
```

Reply:
@jvallesm 
I will modify it!

But, I read the [document](https://pkg.go.dev/github.com/frankban/quicktest#section-readme).
![image](https://github.com/instill-ai/component/assets/64747455/3286e2d7-f9e9-4118-b482-bd2e28fed1f3)

It seems the function for testing.Run and quicktest.Run are similar.
And, even we use `t.Run`, we still can use quicktest for assertion.
Do you know the different between them? Why currently the codebase choose `c.Run` rather than `t.Run`?

Thank you in advance!

Reply:
> It seems the function for testing.Run and quicktest.Run are similar.
> And, even we use t.Run, we still can use quicktest for assertion.
> Do you know the different between them?

[`quicktest.C` is a wrapper over `testing.TB`](https://github.com/frankban/quicktest/blob/master/quicktest.go#L75) so many functions are interchangeable. The reason for defining `c` at the base of your test is saving you the creation of `c` in each subtest (a couple of lines here, more as your test function grows). It also allows you to run, for example, `c.Parallel`. But in this case is consistency about the tooling and conciseness.

> Why currently the codebase choose c.Run rather than t.Run

I proposed quicktest because I find its assertion style useful and readable. I've been using it for years and I'm happy with it so, since no one was against it, I kept using it out of convenience.